src/main/java/com/ael/viner/config/Config.java

```diff
@@ -44,18 +48,27 @@ public class Config {
                 .defineInRange("vineableLimit", 5, 1, Integer.MAX_VALUE);
 
         VINEABLE_BLOCKS = BUILDER
-                .comment("List of blocks/tags that can be vein mined")
+                .comment("List of blocks/tags that can be vein mined. Tags must start with '#'")
                 .defineList("vineableBlocks", Arrays.asList("#minecraft:ores", "#minecraft:logs", "#minecraft:leaves", "#forge:ores"),
                         obj -> obj instanceof String && ((String) obj).matches("^#?[a-z_]+:[a-z_]+$"));
 
+        UNVINEABLE_BLOCKS = BUILDER
+                .comment("List of blocks that will not vein mined. This will override blocks from tags in VINEABLE_BLOCKS")
+                .defineList("unvineableBlocks", ArrayList::new,
+                        obj -> obj instanceof String && ((String) obj).matches("^[a-z_]+:[a-z_]+$"));
+
         BUILDER.pop();
 
         SPEC = BUILDER.build();
     }
 
     @SubscribeEvent
     public static void onLoadComplete(FMLLoadCompleteEvent event) {
-//        upgradeConfig();
```

Comment:
Was curious what this was for, I see now. 

Pretty good idea for user experience!